packages/dev/core/src/DeviceInput/InputDevices/webDeviceInputSystem.ts

```diff
@@ -13,10 +13,37 @@ const MAX_KEYCODES = 255;
 // eslint-disable-next-line @typescript-eslint/naming-convention
 const MAX_POINTER_INPUTS = Object.keys(PointerInput).length / 2;
 
+class InputValue {
+    private _currentValue: number;
+    private _accumulatingValue: number;
+
+    public constructor(defaultValue: number = 0) {
+        this._currentValue = defaultValue;
+        this._accumulatingValue = defaultValue;
+    }
+
+    public set(value: number): void {
+        this._currentValue = value;
+    }
+
+    public get value(): number {
+        return this._currentValue;
+    }
+
+    public accumulate(value: number): void {
+        this._accumulatingValue += value;
+    }
+
+    public tick(defaultValue: number = 0): void {

```

Comment:
Would be great to have those functions documented. What is the use of this function?